torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1991,17 +1993,50 @@ def _backward_impl(ctx, all_args):
                     context = torch._C._DisableAutocast if disable_amp else nullcontext
                     with tracing(saved_context), compile_context(
                         saved_compile_context
-                    ), context(), track_graph_compiling(aot_config, "backward"):
-                        CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                            bw_module, placeholder_list
-                        )
-                        # Maybe save cache entry
-                        if try_save_cache_entry is not None:
-                            try_save_cache_entry(
-                                CompiledFunction.compiled_bw,
-                                fw_metadata,
-                                aot_config,
+                    ), context(), track_graph_compiling(
+                        aot_config, "backward"
+                    ), METRICS_CONTEXT, dynamo_timed(
```

Comment:
Why do you have to explicitly enter `METRICS_CONTEXT` here? Shouldn't `dynamo_timed` take care of it?